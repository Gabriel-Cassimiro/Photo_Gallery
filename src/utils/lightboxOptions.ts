export const Options = {
	settings: {},
	caption: {},
	buttons: {},
	thumbnails: {},
	progressBar: {}
}
