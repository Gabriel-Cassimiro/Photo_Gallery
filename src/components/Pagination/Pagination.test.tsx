/**
 * @jest-environment jsdom
 */

import { render } from "@testing-library/react"
import { TestProvider } from "../../tests/utils"

import { Pagination } from "./index"

describe("Test Pagination component", () => {
	function Element() {
		return (
			<TestProvider>
				<Pagination />
			</TestProvider>
		)
	}

	it("Should render correctly", () => {
		const { container } = render(<Element />)
		expect(container).toMatchSnapshot()
	})
})
