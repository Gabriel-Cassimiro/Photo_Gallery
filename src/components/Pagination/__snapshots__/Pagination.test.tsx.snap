// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Pagination component Should render correctly 1`] = `
<div>
  <div
    class="flex justify-center my-2 gap-2"
  >
    
    
  </div>
</div>
`;
